packages/twilio-run/package.json

```diff
@@ -53,14 +53,15 @@
     "cosmiconfig": "^7.0.0",
     "debug": "^3.1.0",
     "dotenv": "^6.2.0",
+    "enquirer": "^2.3.6",
```

Comment:
I see this is added but nothing in this PR uses it?

Reply:
Ah, I see your note about the typedoc. I'll resolve this since you addressed it already :)

Reply:
It's a peer dependency of listr2 that was giving issues if not installed separately